packages/client/wallets/aa/src/blockchain/wallets/EVMAAWallet.ts

```diff
@@ -117,38 +115,35 @@ export class EVMAAWallet extends LoggerWrapper {
         });
     }
 
-    //TODO @matias: review this method.
-    // First, I would like to use TransactionRequest from viem instead of ethers.
-    // Second, we need to check if chain supports eip-1559 ro not:
```

Comment:
If you don't send the params `kernelClient.sendTransaction` will figure it out and handle for you.

Reply:
More extensive testing will be included the passkey release QA